lib/liveValidation/liveValidator.ts

```diff
@@ -225,6 +237,10 @@ export class LiveValidator {
       const spec = await this.getSwaggerInitializer(this.loader!, swaggerPath);
       if (spec !== undefined) {
         allSpecs.push(spec);
+        if (this.options.enableRoundTripValidator) {
```

Comment:
Can you move operationLoad.init outside of while loop and pass in `allSpecs`?

Reply:
I need the file name to obtain the provider namespace.